vendor/github.com/coreos/etcd/embed/etcd.go

```diff
@@ -160,10 +160,7 @@ func StartEtcd(inCfg *Config) (e *Etcd, err error) {
 			return srv.Serve(e.Peers[i].Listener)
 		}
 		e.Peers[i].close = func(ctx context.Context) error {
-			// gracefully shutdown http.Server
-			// close open listeners, idle connections
-			// until context cancel or time-out
-			return srv.Shutdown(ctx)
+			return nil
```

Comment:
I can't see how it's possible to have an analogue to this, because it was net new addition before.    I looked and can't see an equivalent other than the existing code to stop etcd.